config/samples/external_access/cert-manager-certificate.yaml

```diff
@@ -0,0 +1,17 @@
+---
+apiVersion: cert-manager.io/v1alpha2
+kind: Certificate
+metadata:
+  name: cert-manager-certificate
+  namespace: <your-namespace>
```

Comment:
[nit] I had a quick look at other files in the `samples` dir and it looks like we generally do not specify an entry for `namespace` at all. To be consistent I'd say we should do the same in all files in this PR

Reply:
Good point. Resolved in latest commit.